### DISCUSSION TITLE
Using a database type wxSQLite3, is it possible to express not literal keys?

### DISCUSSION THREAD

**Discussion by rossanoparis** [pawelsalawa/sqlitestudio#5175] [General]
Referring the link below, which is related to this repository: [SQLite3MultipleCiphers](https://github.com/utelle/SQLite3MultipleCiphers)
[SQLite3MultipleCiphers](https://utelle.github.io/SQLite3MultipleCiphers/docs/configuration/config_sql_pragmas/#key-handling)

How could I express a password key like this one:
![image](https://github.com/user-attachments/assets/c6afa5cf-1123-4fd6-9533-dbcf06925503)

Is it possible actually by using this interface?
It seams that only literal key are admitted
![image](https://github.com/user-attachments/assets/4558daec-85ea-44e1-af57-fc590162e6d4)

Thank you, regards

**Comment by pawelsalawa** [comment]
Leave the password field empty (it won't be used then) and pass the full `PRAGMA key = 'raw:....` in the "Cipher configuration" field below.

**Comment by rossanoparis** [comment]
Thank you @pawelsalawa for your useful tip

Only for those who will step on this thread, I report the picture below with the correct parameterization in order to access a DB with a key expressed like the "example 3" described here [SQLite3MultipleCiphers](https://utelle.github.io/SQLite3MultipleCiphers/docs/configuration/config_sql_pragmas/#key-handling)

![image](https://github.com/user-attachments/assets/b5b040de-f908-412b-a368-6ee53c255de1)
